### PR TITLE
Add Jörmungandr integration tests

### PR DESCRIPTION
**Pull request by Anviking**
# Issue Number

#242 

# Overview

- [x] Added integration test suite
- [x] Added `jormungandr` config
- [x] Check that `jormungandr` launches

# Comments

### CODE REVIEW

File: lib/jormungandr/test/data/jormungandr/node.config [hunk 1]
```diff
@@ -0,0 +1,10 @@
+storage: "/tmp/cardano/storage"
```
Comment (Anviking) [inline lib/jormungandr/test/data/jormungandr/node.config:1]: Have to make sure this exists when using it for real like for the http-bridge https://github.com/input-output-hk/cardano-wallet/blob/fb2110589405fd4431fe488b563679c8f404723a/lib/http-bridge/test/integration/Main.hs#L81-L85

Comment (KtorZ) [reply lib/jormungandr/test/data/jormungandr/node.config:1]: I'd suggest something like `/tmp/cardano-wallet-jormungandr` to be more consistent with the bridge. Also, yes, the directory should probably be created. Not sure how Jormungandr handles it.

**Review by KtorZ — APPROVED** [review]
Looks good for now :+1: 
Looking forward to making our first API calls to it :D

**Comment by KtorZ** [comment]
:warning: please don't merge manually :warning: 

Trying bors today...

**Pull request merged**